src/panels/Menu/hooks.ts

```diff
@@ -2,7 +2,10 @@ import { useCallback, useEffect } from 'react';
 
 import { useGetBoolPropertyValue } from '@/api/hooks';
 import { useAppDispatch, useAppSelector } from '@/redux/hooks';
-import { setMenuItemEnabled, setMenuItemVisible } from '@/redux/local/localSlice';
+import {
+  setMenuItemEnabled,
+  setMenuItemVisible as setMenuItemVisibleRedux
```

Comment:
Why do we need the Redux here? all actions that are dispatched go to redux

Reply:
There is a naming conflict with the action `setMenuItemVisible` and a function that is being returned by the `useMenuItems` hook